src/test/java/com/alkemy/ong/mapper/TestimonialMapperTest.java

```diff
@@ -0,0 +1,134 @@
+package com.alkemy.ong.mapper;
+
+
+import com.alkemy.ong.dto.TestimonialDto;
+import com.alkemy.ong.dto.TestimonialDtoFull;
+import com.alkemy.ong.entity.TestimonialEntity;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.mapstruct.factory.Mappers;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class TestimonialMapperTest {
+
+    private static TestimonialMapper mapper;
+
+    @BeforeAll
+    public static void setUp() {
+        mapper= Mappers.getMapper(TestimonialMapper.class);
+
+    }
+    @Test
+    void testimonialDtoToTestimonialEntity() {
+        TestimonialDto dto= new TestimonialDto();
+        dto.setImage("img.jpg ");
+        dto.setName("Name Testimony");
+        dto.setContent("Content");
+        TestimonialEntity ent= mapper.testimonialDtoToTestimonialEntity(dto);
+        System.out.println(ent.getName());
+        assertAll(
+                ()->{
+                    assertEquals(dto.getName(),ent.getName());
+                    assertEquals(dto.getImage(), ent.getImage());
+                    assertEquals("Content",ent.getContent());
+                }
+        );
+    }
+    @Test
+    public void testimonyDtoMapperTestSimpleNull(){
+        TestimonialDto dto=null;
+        TestimonialEntity entity=mapper.testimonialDtoToTestimonialEntity(dto);
+        assertEquals(null,entity);
+    }
+
+    @Test
+    void testimonyEntityToTestimonyDto() {
+        TestimonialEntity testimonial= new TestimonialEntity();
+        testimonial.setId("1234");
+        testimonial.setImage("testimonial.jpg");
+        testimonial.setName("name testimonial");
+        testimonial.setSoftDelete(false);
+        testimonial.setContent("Content");
+        testimonial.setTimestamps(LocalDateTime.now());
+
+        TestimonialDto dto=mapper.testimonialToTestimonialDto(testimonial);
+        System.out.println(dto.getName());
+        assertAll(
+                ()->{
+                    assertEquals(dto.getName(),testimonial.getName());
+                    assertEquals(dto.getImage(), testimonial.getImage());
+                    assertEquals("Content",testimonial.getContent());
+                }
+        );
+    }
+    @Test
+    public void testimonialMapperTestSimpleNull(){
+        TestimonialEntity entity=null;
+        TestimonialDto dto=mapper.testimonialToTestimonialDto(entity);
```

Comment:
You can eliminate one row straight using null like this:
 TestimonialDto dto=mapper.testimonialToTestimonialDto(null);
        assertEquals(null,dto);